viewer/constant.js

```diff
@@ -4,13 +4,35 @@
     angular.module('chaise.viewer')
 
     .constant('viewerConstant', {
```

Comment:
Could you add a comment explaining why this is hard coded (I know the app is only used by RBK).  Maybe we should discuss a way to expand this to be customizable in the future.

Reply:
We're still not sure how this will be used in other deployments. So for now, @hongsudt suggested using something inside the viewer code but centralized that can later be moved to a configuration file. Since we were not sure we want to reuse the chaise-config method, or introduce a new method for this; I just decided to add a constant in the code.

I will add this comment to the code as well.